secret/scanner.go

```diff
@@ -72,6 +141,34 @@ func (s Scanner) Scan(args ScanArgs) types.Secret {
 	}
 }
 
+func removeAllowedSecrets(allRuleResults []RuleResult, allowedLocations [][]int) []RuleResult {
+	if len(allowedLocations) > 0 {
+		results := make([]RuleResult, 0)
+		for _, result := range results {
+			found := false
+			if result.EndPosition < allowedLocations[0][0] {
+				results = append(results, result)
+				continue
+			}
```

Comment:
Looks like we don't need these lines since we do the same thing at 154-156 lines.